qualtran/Controlled.ipynb

```diff
@@ -54,9 +58,56 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from qualtran import CtrlSpec\n",
+    "from qualtran import CtrlSpec, QUInt, QBit\n",
     "\n",
-    "ctrl_spec = CtrlSpec(cvs=[0, 1, 0, 1])\n",
+    "ctrl_spec = CtrlSpec(qdtypes=QUInt(4), cvs=0b0110)\n",
+    "assert ctrl_spec.is_active(6)\n",
```

Comment:
this should match the example above where you explicitly note that `0b0101` is integer `5`